src/MDMSpringTimingCurve.m

```diff
@@ -30,8 +30,7 @@ @implementation MDMSpringTimingCurve {
 @synthesize tension = _tension;
 
 - (instancetype)init {
-  [self doesNotRecognizeSelector:_cmd];
-  return nil;
+  return [self initWithMass:1 tension:341.023716023995 friction:36.9336548976131];
```

Comment:
Please cast these double literals to (CGFloat) for 32-bit arch support.

Reply:
Done.